third_party/move/tools/move-mutator/src/operators/literal.rs

```diff
@@ -0,0 +1,222 @@
+use crate::operator::{MutantInfo, MutationOperator};
+use crate::report::{Mutation, Range};
+use move_command_line_common::files::FileHash;
+use move_compiler::expansion::ast;
+use move_compiler::expansion::ast::Value_;
+use std::fmt;
+use std::fmt::Debug;
+
+/// Break and continue mutation operator.
+/// Replaces break and continue statements with each other or deletes them.
+#[derive(Debug, Clone)]
+pub struct Literal {
+    operation: ast::Value,
+}
+
+impl Literal {
+    /// Creates a new instance of the break/continue mutation operator.
+    #[must_use]
+    pub fn new(operation: ast::Value) -> Self {
+        Self { operation }
+    }
+}
+
+impl MutationOperator for Literal {
+    fn apply(&self, source: &str) -> Vec<MutantInfo> {
+        let start = self.operation.loc.start() as usize;
+        let end = self.operation.loc.end() as usize;
+        let cur_op = &source[start..end];
+
+        // Group of exchangeable break/continue statements.
+        let ops: Vec<&str> = match &self.operation.value {
+            Value_::Address(_addr) => {
+                vec![
+                    "0x0",
+                    "0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF",
+                ]
+            },
+            Value_::Bool(_bool_val) => {
+                vec!["true", "false"]
+            },
+            Value_::Bytearray(_bytearray) => {
+                vec!["b\"121112\""]
+            },
+            Value_::U8(_u8_val) => {
+                vec!["0u8", "255u8"]
+            },
+            Value_::U16(_u16_val) => {
+                vec!["0u16", "65535u16"]
```

Comment:
Maybe something like this could be done in the future? Since we have many possibilities here
```suggestion
            Value_::U16(u16_val) => {
                vec!["0u16", "65535u16", (u16_val + 1).to_string(), (u16_val - 1).to_string()]
```

Reply:
I've added it now.
It changed the main vector type and I'd to add `.to_owned()` to the literals, but now it covers also such tests.